src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -450,22 +616,22 @@ void M(Base b, Derived d)
     }
 }
 ";
-            var comp = CreateCompilationWithoutCovariantReturns(source).VerifyDiagnostics(
+            var comp = CreateCompilationWithCovariantReturns(source, parseOptions: TestOptions.WithoutCovariantReturns).VerifyDiagnostics(
                 // (8,28): error CS8652: The feature 'covariant returns' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
                 //     public override string M() => null;
                 Diagnostic(ErrorCode.ERR_FeatureInPreview, "M").WithArguments("covariant returns").WithLocation(8, 28)
                 );
             verify(SourceView(comp, assignments));
 
             // Test against a runtime that does not admit support for covariant returns.
-            comp = CreateCompilationWithoutCovariantReturns(source, pretendRuntimeSupportsCovariantReturnsFeature: false).VerifyDiagnostics(
+            comp = CreateCompilationWithoutCovariantReturns(source).VerifyDiagnostics(
                 // (8,28): error CS8778: 'Derived.M()': Target runtime doesn't support covariant return types in overrides. Return type must be 'object' to match overridden member 'Base.M()'
                 //     public override string M() => null;
                 Diagnostic(ErrorCode.ERR_RuntimeDoesNotSupportCovariantReturnsOfClasses, "M").WithArguments("Derived.M()", "Base.M()", "object").WithLocation(8, 28)
                 );
             verify(SourceView(comp, assignments));
 
-            comp = CreateCompilationWithCovariantReturns(source, pretendRuntimeSupportsCovariantReturnsFeature: false).VerifyDiagnostics(
+            comp = CreateCompilationWithoutCovariantReturns(source, parseOptions: TestOptions.WithCovariantReturns).VerifyDiagnostics(
```

Comment:
>WithCovariantReturns [](start = 94, length = 20)

Could you please remind what is special about these options?

Reply:
Is this just about the language version?

---
In reply to: [452420961](https://github.com/dotnet/roslyn/pull/45824#discussion_r452420961) [](ancestors = 452420961)

Reply:
Yes, `TestOptions.WithCovariantReturns` is a set of parse options for a language version in which covariant returns being supported. This particular line is testing an attempt to use the feature on a runtime that does not support it.

---
In reply to: [452422125](https://github.com/dotnet/roslyn/pull/45824#discussion_r452422125) [](ancestors = 452422125,452420961)